### PR TITLE
Added new transition, and new status for Access Cards.

### PR DESCRIPTION
**Pull request by lukaszkarykowski**

### CODE REVIEW

File: src/ralph/access_cards/models.py [hunk 1]
```diff
@@ -25,7 +25,8 @@ class AccessCardStatus(Choices):
     damaged = _('damaged')
     used = _('in use')
     free = _('free')
-    return_in_progress = _('return in progres')
+    reserved = _("reserved")
+    return_in_progress = _('return in progress')
```
Comment (romcheg) [inline src/ralph/access_cards/models.py:28]: Always add new statuses to the end of the list.

**Review by romcheg — CHANGES_REQUESTED** [review]

File: src/ralph/access_cards/models.py (hunk 1 above)
Comment (lukaszkarykowski) [reply src/ralph/access_cards/models.py:28]: ok. Done.

**Review by romcheg — COMMENTED** [review]
Looks like you have to generate migrations to make CI happy again. LGTM otherwise.

**Review by szymi- — APPROVED** [review]

**Review by romcheg — APPROVED** [review]
LGTM, thanks for following up!

**Pull request merged**